adapters/smartyads/smartyadstest/params/race/banner.json

```diff
@@ -0,0 +1,5 @@
+{
+  "host": "smartyadstest.com",
```

Comment:
Doesn't really matter since it is just an arbitrary string, but now that the host field is actually the host prefix, would be nice to make the examples match. This way people viewing these tests will not get confused as what is expected in the host field.

Reply:
Hello @hhhjort ! Sure I have fixed it ! Thank you for your attention!